cupy/random/distributions.py

```diff
@@ -390,6 +390,32 @@ def poisson(lam=1.0, size=None, dtype=int):
     return x
 
 
+def rayleigh(scale=1.0, size=None, dtype=float):
+    """Rayleigh distribution.
+
+    Returns an array of samples drawn from the rayleigh distribution.
+    Its probability density function is defined as
+
+      .. math::
+         f(x) = \\frac{x}{scale^2}e^{\\frac{-x^2}{2-scale^2}},
```

Comment:
Could you fix the multiple-letter name `scale` of the variable?  (Please rename it or fix its TeX.)